frontend/src/services/web3-provider/ethereum-provider.ts

```diff
@@ -66,13 +67,17 @@ export abstract class EthereumProvider extends EventEmitter implements IEthereum
         chainName: name,
         rpcUrls: [rpcUrl],
         blockExplorerUrls: [explorerUrl],
-        nativeCurrency: {
-          name: 'Ethereum',
-          symbol: 'ETH',
-          decimals: 18,
-        },
+        nativeCurrency: nativeCurrency
+          ? toRaw(nativeCurrency)
+          : {
+              name: 'Ethereum',
```

Comment:
The name of the nativeCurrency is actually Ether and not Ethereum.

Reply:
True!